### ISSUE TITLE
signing a key on the commandline never completes

### ISSUE THREAD

**Issue by anarcat** [gnome-keysign/gnome-keysign#76]
Using the 1.0.1 debian package (self-built because not officially released yet), I cannot seem to sign a key properly. I have tried this:

```
$ gpg --export C4BC2DDB38CCE96485EBE9C2F20691179038E5C6 > dkg.gpg
$ gnome-keysign-sign-key dkg.gpg 
WARNING:keysign.gpgmeh:INV_SGNR: '9 7D4F37B1948ADAFF'
WARNING:keysign.gpgmeh:INV_SGNR: '9 586073B34023702F'

```

Then that pops open two terminal windows running the following command:

```
mutt mailto:dkg@debian.org?&subject=Your%20signed%20key%20C4BC2DDB38CCE96485EBE9C2F20691179038E5C6&body=Hi%20Daniel%20Kahn%20Gillmor%20%3Cdkg%40debian.org%3E%2C%0A%0A%0AI%20have%20just%20signed%20your%20key%0A%0A%20%20%20%20%20%20C4BC2DDB38CCE96485EBE9C2F20691179038E5C6%0A%0A%0AThanks%20for%20letting%20me%20sign%20your%20key%21%0A%0A--%0AGNOME%20Keysign%0A&attachment=/tmp/gnome-keysign-rn1qyx4m.asc
```

But while mutt normally is able to send email, for some reason, when it's from gnome-keysign, that doesn't work at all: mutt returns (strangely without sending any email) and gnome-keysign just hangs there. Interrupting with <kbd>control-c</kbd> yields the following traceback:

```
Traceback (most recent call last):
  File "/usr/bin/gnome-keysign-sign-key", line 11, in <module>
    load_entry_point('gnome-keysign==1.0.1', 'console_scripts', 'gnome-keysign-sign-key')()
  File "/usr/lib/python3/dist-packages/keysign/SignKey.py", line 45, in main
    input()
KeyboardInterrupt
```

It seems the program deliberately be waiting for my input, but never tells me so. Looking at the source, it seems that this might be a logging level problem. But looking at the [logging tutorial](https://docs.python.org/3/howto/logging.html#logging-basic-tutorial), I would recommend explicitely using `print` for such prompts instead of the logging system.

**Comment by anarcat** [comment]
On 2019-01-21 14:30:01, muelli wrote:
> oh, interesting.
> Yeah, a print sounds more clever there. I wonder why we made it a
> logging statement in first place. Seems silly now.

I've done the same mistake before. :)

> In fairness, the CLI app is rather auxiliary; but we still ought to make it as nice as possible.

heh. I'm trying out the monkeysign replacements, and commandline use is
one of my requirements. ;)

`--debug` would certainly be useful, for example...

**Comment by muelli** [comment]
Thanks again for the report. Should be fixed with 0fc5abb.
